CHANGELOG.md

```diff
@@ -63,7 +65,9 @@ The document follows the conventions described in [“Keep a CHANGELOG”](http:
 - default constructor now has an optional `node` argument which is used to consruct the object e.g. `new SVG.Rect(rectNode)`
 - SVG.Elements constructor now tries to import svgjs:data from the node
 - `SVG.on()` calls the listener in the context of the passed object. el.on always uses the svg.js object as context
-- `SVG.on()/off()` and `el.on()/off()` now accepts multiple comma or space seperated events e.g. "mousedown, foo bar" (#727)
+- `SVG.on()/off()` and `el.on()/off()` now accepts multiple comma or space separated events e.g. "mousedown, foo bar" (#727)
+- Matrices now apply transformations like `scale`, `translate`, etc... by left multiplying them to simplify transformations
+- Transforming an element is now much simpler
```

Comment:
maybe better write something like "The way `transform()` works had been completely reworked. See the docs for further reference"

Reply:
Did it!